test/test_cuda.py

```diff
@@ -5169,6 +5179,28 @@ def test_graph_grad_scaling(self, device, dtype, optim_info, foreach, fused):
             self.assertEqual(scaler._growth_tracker, growth_tracker)
 
 
+class TestGDS(TestCase):
+    # can't run in CI due to lack of ext4/xfs mount on CI docker containers
```

Comment:
Can you detect the filesystem and make this a real test so that anyone running on a machine where this could work, would work.

If we do it this way, we would have to modify this file every time we want to run this test.